### PR TITLE
Add the option to display the Account Estimated Earnings on BTC or in USD

### PR DESCRIPTION
**Pull request by prfiredragon**
I add the option to display the Account Estimated Earnings on BTC or in USD. The default is BTC.

### CODE REVIEW

File: lendingbot.py [hunk 1]
```diff
@@ -129,10 +133,14 @@
...
 		coincfg[cur[0]] = dict(minrate=(Decimal(cur[1]))/100, maxactive=Decimal(cur[2]))
+if args.nobtc:
+	onBTC = False
+else:
+	onBTC = True
 #End handling args.
 
 #Check if we need a config file at all (If all settings are passed by args, we won't)
-if args.apikey and args.apisecret and args.sleeptimeactive and args.sleeptimeinactive and args.mindailyrate and args.maxdailyrate and args.spreadlend and args.gapbottom and args.gaptop and args.sixtydaythreshold:
+if args.apikey and args.apisecret and args.sleeptimeactive and args.sleeptimeinactive and args.mindailyrate and args.maxdailyrate and args.spreadlend and args.gapbottom and args.gaptop and args.sixtydaythreshold and args.nobtc:
```
Comment (Evanito) [inline lendingbot.py]: Don't "require" an arg that has toggleable functionality.

File: lendingbot.py [hunk 2]
```diff
@@ -283,8 +292,14 @@ def cancelAndLoanAll():
...
-				msg = bot.cancelLoanOffer(cur,offer['id'])
-				log.cancelOrders(cur, msg)
+				try:
+					msg = bot.cancelLoanOffer(cur,offer['id'])
+					log.cancelOrders(cur, msg)
+				except Exception as e:
+					log.log("Error canceling loan offer: " + str(e))
+
+				#msg = bot.cancelLoanOffer(cur,offer['id'])
+				#log.cancelOrders(cur, msg)
```
Comment (Evanito) [inline lendingbot.py]: Does this need to be changed?

Have you experienced errors with this code? It seems bundled in with the USD feature.

File: lendingbot.py (hunk 1 above)
Comment (prfiredragon) [inline lendingbot.py]: done.

File: lendingbot.py (hunk 2 above)
Comment (prfiredragon) [inline lendingbot.py]: I was testing (https://github.com/Mikadily/poloniexlendingbot/pull/107), I already remove it.

**Comment by rnevet** [comment]
I would like to have a more generic approach, where we can offer to switch the currency in the html. Someone might be more interested in ETH or EUR (if available).

**Comment by rnevet** [comment]
And I wouldn't name USDT -> USD, not exactly the same and no reason to.

**Comment by Evanito** [comment]
That is a very good idea, you can change the config option to
```
outputCurrency = BTC #Options are BTC, USDT, ETH or anything as long as it has a direct BTC market.
```
Doing any other currencies that don't have a direct Poloniex market (Such as EUR) will require more overhead and we would need to agree on a source api, I recommend Preev.

**Comment by rnevet** [comment]
Lets stick to Polo markets then.

**Comment by prfiredragon** [comment]
Ok. Good idea, I am going began to test with BTC, USDT, ETH, XMR and DOGE.

**Comment by rnevet** [comment]
Cool!

**Comment by prfiredragon** [comment]
That was interesting. Done. The options are BTC, USDT and all in the BTC market.

**Comment by Evanito** [comment]
I have reviewed the code and everything seems to be in order. Though with these kinds of changes I will want some user testing before merge. I will do it tomorrow if Raanan doesn't beat me to it. 
Great job and cheers!

**Comment by prfiredragon** [comment]
Thank You. You are welcome.

**Pull request merged**